index.html

```diff
@@ -42,6 +42,82 @@ <h1 class="headline">I'm Didier<br/>Glad you're here!</h1>
+                    </div>
+                <div class="img-container">
+                    <img src="images/work1.png"  class="work-image" alt="Showcase of the work"> 
+                </div>
+                <div class="work-details-popup">
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tags-button">
+                        <ul class="tags">
+                            <li class="tag">html</li>
+                            <li class="tag">css</li>
+                            <li class="tag">javascript</li>
+                        </ul>
+                    </div>
+                    <div class="buttons">
+                        <a href="#" class="work-link">
+                            See Live
+                            <img src="images/live.png" alt="arrow">
+                        </a>
+                        <a href="#" class="work-link">
+                            See Source
+                            <img src="images/github.png" width="20" height="20" alt="arrow">
+                        </a>
+                    </div>
+                </div>   
+            </div>
+        </div>
+    </div>
+    <!--Popup to show works-->
+    <div class="popup isHidden">
+        <div class="header-popup">
+            <h2 class="work-title-popup">Tonic</h2>
+            <img class="close-top-desktop" src="images/close-black.png" alt="Close">
+        </div>
+        <div class="work-subtitle">
+            <p class="company">Canopy</p>
+            <img src="images/counter.png" width="10" height="10" alt="counter">
+            <p class="company-title">Back End Dev</p>
+            <img src="images/counter.png" alt="counter">
+            <p class="company-year">2015</p>
+        </div>
+        <div class="img-container-popup">
+            <img src="images/snapshot.png" class="work-image-popup" alt="Showcase of the work"> 
+        </div>
+        <div class="description-overall">
+            <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags-button">
+                <ul class="tags">
+                    <li class="tag">html</li>
+                    <li class="tag">css</li>
+                    <li class="tag">javascript</li>
+                </ul>
+                <a href="#" class="work-link">
+                    See Live
+                    <img src="images/live.png" alt="arrow">
+                </a>
+                <a href="#" class="work-link">
+                    See Source
+                    <img src="images/github.png" width="20" height="20" alt="arrow">
+                </a>
+            </div>
+        </div>
+    </div>
     <section  class="works-mobile" id="portfolio-mobile">
         <div class="work-container">
             <div class="work">
```

Comment:
- [ ] As requested in the project requirements, The projects section should be created dynamically using Javascript. all already created HTML code for the projects section list should be removed, and instead, it should be added via JavaScript. You should only have a container for the section in the HTML file.